### DISCUSSION TITLE
使用截图OCR功能后，用剪贴板中的内容去重命名文件总是有特殊符号的问题

### DISCUSSION THREAD

**Discussion by ghost** [shuax/MouseInc.Settings#121] [General]
使用截图OCR功能后，特别是多行文字，用剪贴板中的内容去重命名文件总是有特殊符号的问题，希望能加相关设置项，改善这项功能。

**Comment by shuax** [comment]
ocr和重命名本来没啥关系，不太可能搞个设置。

**Reply by littlegolden** [reply]
> ocr和重命名本来没啥关系，不太可能搞个设置。

可以搞一个「合并为一行 + 完全 trim」的功能
